code/modules/surgery/organs/pain.dm

```diff
@@ -31,9 +35,12 @@
 
 // message is the custom message to be displayed
 /mob/living/carbon/human/proc/custom_pain(message)
-	if(stat >= UNCONSCIOUS)
-		return
-
+	if(stat == (DEAD | ANESTHETIZED))
+		return				//No messages when Anesthetized or Dead
+	if(status_flags & FAKEDEATH)
```

Comment:
So` custom_pain(message)` wakes people out of FAKEDEATH. Folks should use FAKEDEATH sparingly, ya? And if you use FAKEDEATH and suddenly get beaten on while fakedead, uh, that sounds like self-antagging by your attacker. If you get woken out of FakeDeath from pain, that's a bit wonky! Maybe there's another check in `WakeUp`, but I wanted to be extra sure.